invenio_vocabularies/factories.py

```diff
@@ -137,6 +142,39 @@ def get_service(self):
         raise NotImplementedError("Service not implemented for EDMO Affiliations")
 
 
+class SubjectsEuroSciVocVocabularyConfig(VocabularyConfig):
+    """EuroSciVoc Subjects Vocabulary Config."""
+
+    config = euroscivoc_ds_config
+    vocabulary_name = "subjects:euroscivoc"
+
+    def get_service(self):
+        """Get the service for the vocabulary."""
+        raise NotImplementedError("Service not implemented for EuroSciVoc Subjects")
+
+
+class SubjectsGEMETVocabularyConfig(VocabularyConfig):
+    """GEMET Subjects Vocabulary Config."""
+
+    config = gemet_ds_config
+    vocabulary_name = "subjects:gemet"
+
+    def get_service(self):
+        """Get the service for the vocabulary."""
+        raise NotImplementedError("Service not implemented for GEMET Subjects")
+
+
+class SubjectsBODCVocabularyConfig(VocabularyConfig):
+    """BODC Subjects Vocabulary Config."""
+
+    config = bodc_ds_config
+    vocabulary_name = "subjects:bodc"
```

Comment:
```suggestion
    vocabulary_name = "subjects:bodc-puv"
```

nit: just to be very specific in case we import different parts later on.